ast/misc.go

```diff
@@ -2539,7 +2539,7 @@ func (n *TableOptimizerHint) Restore(ctx *format.RestoreCtx) error {
 	switch n.HintName.L {
 	case "max_execution_time":
 		ctx.WritePlainf("%d", n.HintData.(uint64))
-	case "tidb_hj", "tidb_smj", "tidb_inlj", "hash_join", "merge_join", "inl_join", "bc_join", "tidb_bcj":
+	case "tidb_hj", "tidb_smj", "tidb_inlj", "hash_join", "merge_join", "inl_join", "bc_join", "bcj_local", "tidb_bcj":
```

Comment:
Nit: according to https://www.abbreviations.com/abbreviation/BROADCAST, seems `BRDCST` is a proper abbreviation for `broadcast`

Reply:
It seems brdcst is for media, I think it's quite verbose. Can we make it brief and easy to write ?

Reply:
if you shorten it to "brdcst" i'd rather you just spell it out entirely

```sql
select /*+ broadcast_join(...) */ ...
```

Reply:
@hanfei1991 @lzmhhh123 PTAL the comment from @kennytm

Reply:
I support the `bc_join` as the abbreviation.